_docs/faqs.md

```diff
@@ -22,6 +22,8 @@ Minecraft Dev supports all of the major Java Minecraft development platforms. Th
 If you find a problem with the plugin, please report it to our [issue tracker](https://github.com/minecraft-dev/MinecraftDev/issues).
 We try to deal with bugs as quickly as possible and take the quality of this plugin very seriously.
 
+In some cases, you may be able to report an exception related to the Minecraft Dev plugin directly through IntelliJ. If you choose to do this, the reported stack traces could include sensitive information such as package names or source code. This data will be publicly accessible and searchable through the issue tracker.
```

Comment:
How about saying unobfuscated class/member names and/or small fragments of source code?

Reply:
Saying "small" could understate how much source code is actually revealed, which could lead to an inaccurate risk assessment for people who are using this plugin while working on private projects.

In some cases, the amount of source code included can be substantial. As an example, https://github.com/minecraft-dev/MinecraftDev/issues/1393#issuecomment-986181603 leaks 106 lines of what is presumably private code.

Reply:
Oh wow I was not aware of this, okay